### PR TITLE
fix: fix lab service process sample

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA Link
[https://blocksphere2020.atlassian.net/browse/DBIO-848](url)
[https://blocksphere2020.atlassian.net/browse/DBIO-897](url)
[https://blocksphere2020.atlassian.net/browse/DBIO-899](url)
[https://blocksphere2020.atlassian.net/browse/DBIO-998](url)
[https://blocksphere2020.atlassian.net/browse/DBIO-999](url)
[https://blocksphere2020.atlassian.net/browse/DBIO-1084](url)
[https://blocksphere2020.atlassian.net/browse/DBIO-1206](url)

### Changelog / Description

- [x] add biolical sample input field on lab registration service
- [x] fix dorpdown service category link. (this dropdown now can get data from database)
- [x] remove covid-19 from type biological sample
- [x] add service category field on lab registration add service
- [x] add service category field on lab registration detail/edit service
- [x] if biological sample type = covid, on process test page, skipped to wetwork process. (not process QC)
- [x] add service category field on lab add service

### CODE REVIEW

File: src/views/Dashboard/Lab/result.vue [hunk 1]
```diff
@@ -121,6 +121,7 @@ import sendTransaction from '../../../lib/send-transaction'
 import localStorage from '../../../lib/local-storage'
 import Wallet from '../../../lib/dgnx-wallet'
 import { mapState } from 'vuex'
+import { SUCCESS } from '@/constants/specimen-status'
```
Comment (luthfiazhari) [inline src/views/Dashboard/Lab/result.vue:124]: missing variable on speciment status

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue [hunk 2]
```diff
@@ -65,16 +66,45 @@ export default {
...
-          this.api,
-          this.pair,
-          this.specimenNumber,
-          "Arrived",
-          () => {
-            this.isLoading = false
-            this.specimenAlertDialog = true
-          }
-        )
+        console.log(this.specimenNumber)
```
Comment (hilyds) [inline src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue:69]: please remove this line

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue [hunk 3]
```diff
@@ -65,16 +66,45 @@ export default {
...
+        console.log(this.isBiological, "biological")
```
Comment (hilyds) [inline src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue:70]: please remove this line

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue [hunk 4]
```diff
@@ -65,16 +66,45 @@ export default {
...
+            this.specimenNumber,
+            "QualityControlled",
+            () => {
+              this.isLoading = false
+              this.specimenAlertDialog = true
+            }
+          )
+        }
+
+        // if (this.isBiological == true) {
```
Comment (hilyds) [inline src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue:96]: and also remove these lines

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/index.vue [hunk 5]
```diff
@@ -188,6 +192,18 @@ export default {
         this.$router.push({ name: 'lab-dashboard-order-history' })
       }
       const order = await getOrdersDetail(this.api, this.orderId)
+      const serviceInfo = await queryServicesById(this.api, order.serviceId)
+
+      this.dnaCollectionProcess = serviceInfo.info.dnaCollectionProcess
+      if (this.dnaCollectionProcess.includes("Covid")) {
+        this.isBiological = true
+      }
+      // this.isBiological = true//just for testing deleted soon
```
Comment (hilyds) [inline src/views/Dashboard/Lab/OrderHistory/ProcessOrder/index.vue:201]: also these command lines

File: src/views/Dashboard/Lab/result.vue (hunk 1 above)
Comment (hilyds) [inline src/views/Dashboard/Lab/result.vue:124]: why only 1 status success?

**Review by hilyds — CHANGES_REQUESTED** [review]

**Comment by hilyds** [comment]
is this PR doesn't include https://blocksphere2020.atlassian.net/browse/DBIO-900 ?

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue (hunk 2 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue:69]: done, this line deleted

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue (hunk 3 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue:70]: deleted too

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue (hunk 4 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/OrderHistory/ProcessOrder/ReceiveSpecimen.vue:96]: deleted

File: src/views/Dashboard/Lab/OrderHistory/ProcessOrder/index.vue (hunk 5 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/OrderHistory/ProcessOrder/index.vue:201]: this line deleted too

File: src/views/Dashboard/Lab/result.vue (hunk 1 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/result.vue:124]: this is for dissabled button if speciment status is success.

**Comment by luthfiazhari** [comment]
> is this PR doesn't include https://blocksphere2020.atlassian.net/browse/DBIO-900 ?

the button is still not working, i still working on that

**Comment by hilyds** [comment]
> > is this PR doesn't include https://blocksphere2020.atlassian.net/browse/DBIO-900 ?
> 
> the button is still not working, i still working on that

then please remove the jira link DBIO-900 in the description above

**Review by hilyds — APPROVED** [review]

**Pull request merged**